scenes/enemy-commander/EnemyCommander.gd

```diff
@@ -0,0 +1,39 @@
+tool
+extends Node2D
+
+export(NodePath) var wave_manager_path
+var wave_manager
+export(NodePath) var call_button_path
+var call_button
+
+export(Array) var wave_sequence = [
+#	[time from prev. wave end to to next call button visible, time to wave spawn],
+	[2, 15],
+	[15, 30],
+]
+var wave_counter = 0
+
+
+func _ready():
+	wave_manager = get_node(wave_manager_path)
+	wave_manager.connect("last_enemy_in_actual_wave_spawned", self, "_on_last_enemy_in_actual_wave_spawned")
+	call_button = get_node(call_button_path)
+	call_button.connect("enemy_start_timeout", self, "_on_enemy_start_timeout")
```

Comment:
I'll ask you also one more thing: I would like to rename the `enemy-call-button.gd` signal from `enemy_start_timeout` to `next_wave_requested`, I think it's easier to understand, what do you think?

Reply:
I agree with the change. I saw the review this morning on Twitch and I saw that it was difficult to understand immediately what the signal was for.